BitwardenShared/UI/Vault/VaultItem/CipherItemStateTests.swift

```diff
@@ -131,4 +131,34 @@ class CipherItemStateTests: BitwardenTestCase {
         ]
         XCTAssertEqual(state.collectionsForOwner.map(\.id), ["1", "2"])
     }
+
+    /// `shouldShowLearnNewLoginActionCard` should be `true`, if the cipher is a login type and configuration is `.add`.
+    func test_shouldShowLearnNewLoginActionCard_True() {
+        let cipher = CipherView.loginFixture(login: .fixture(fido2Credentials: [.fixture()]))
+        var state = CipherItemState(cloneItem: cipher, hasPremium: true)
+        state.isLearnNewLoginActionCardEligible = true
+        XCTAssertTrue(state.shouldShowLearnNewLoginActionCard)
+    }
+
+    /// `shouldShowLearnNewLoginActionCard` should be `false`, if the cipher is not a login type.
+    func test_shouldShowLearnNewLoginActionCard_False() {
+        let cipher = CipherView.cardFixture(card: .fixture(
+            code: "123",
+            number: "123456789"
+        ))
+        var state = CipherItemState(cloneItem: cipher, hasPremium: true)
+        state.isLearnNewLoginActionCardEligible = true
+        XCTAssertFalse(state.shouldShowLearnNewLoginActionCard)
+    }
+
+    /// `shouldShowLearnNewLoginActionCard` should be `false`, if the configuration is not `.add`.
+    func test_shouldShowLearnNewLoginActionCard_False_Config() throws {
```

Comment:
⛏️ We've been using camel case when adding additional context/segments to test function names:
```suggestion
    func test_shouldShowLearnNewLoginActionCard_true() {
        let cipher = CipherView.loginFixture(login: .fixture(fido2Credentials: [.fixture()]))
        var state = CipherItemState(cloneItem: cipher, hasPremium: true)
        state.isLearnNewLoginActionCardEligible = true
        XCTAssertTrue(state.shouldShowLearnNewLoginActionCard)
    }

    /// `shouldShowLearnNewLoginActionCard` should be `false`, if the cipher is not a login type.
    func test_shouldShowLearnNewLoginActionCard_false() {
        let cipher = CipherView.cardFixture(card: .fixture(
            code: "123",
            number: "123456789"
        ))
        var state = CipherItemState(cloneItem: cipher, hasPremium: true)
        state.isLearnNewLoginActionCardEligible = true
        XCTAssertFalse(state.shouldShowLearnNewLoginActionCard)
    }

    /// `shouldShowLearnNewLoginActionCard` should be `false`, if the configuration is not `.add`.
    func test_shouldShowLearnNewLoginActionCard_false_config() throws {
```